### PR TITLE
Class 01

### PR DESCRIPTION
**Pull request by jefferyrayrussell**

### CODE REVIEW

**Comment by jefferyrayrussell** [comment]
Worked three hours to upgrade class-01 branch.   Progress made on all aspects.  Work remains on both html and js.   I misunderstand the first day assignment but feel now I am on the right track though a bit behind.  I feel much more confident than I did yesterday.

File: index.html [hunk 1]
```diff
@@ -11,36 +11,44 @@
...
                 published <time pubdate datetime="2000-01-01">Publish Time</time>
               </div>
+
             </header>
-            <section class="article-body"></section>
+            <section class="project-summary"></section>
+
           </article>
         </section>
       </div>
```
Comment (jalleng) [inline index.html:45]: This closing tag seems to be missing it's opening tag.  I think he is probably unnecessary anyhow.

File: index.html [hunk 2]
```diff
@@ -11,36 +11,44 @@
...
+
     </main>
     <footer>
     </footer>
       <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.2/jquery.min.js"></script>
-      <script src="scripts/blogArticles.js"></script>
-      <script src="scripts/article.js"></script
+      <script src="scripts/portfolioProjects.js"></script>
+      <script src="scripts/app.js"></script
```
Comment (jalleng) [inline index.html:52]: These script tags should not be indented further than the footer.  And this last one is missing a bracket.

File: index.html [hunk 3]
```diff
@@ -11,36 +11,44 @@
     <main>
         <header>
```
Comment (jalleng) [inline index.html:12]: All of the elements inside of main are indented one level too far.

**Pull request merged**